charts/argo/values.yaml

```diff
@@ -36,6 +36,8 @@ controller:
   image:
     # Overrides .images.tag if defined.
     tag: ""
+  # parallelism dictates how many workflows can be running at the same time
+  parallelism: 10
```

Comment:
We should not provide a restrictive default here, can we make the default null and then conditionally set the `paralellism` option in the config-map.

Reply:
Done